src/main/java/org/spongepowered/mod/entity/SpongeHorseColor.java

```diff
@@ -32,4 +34,14 @@ public SpongeHorseColor(int color, String name) {
         super(color, name);
     }
 
+    @Override
+    public DataContainer toContainer() {
+        return (DataContainer)this;
```

Comment:
1) [`DataSerializable#toContainer()](https://github.com/SpongePowered/SpongeAPI/blob/master/src/main/java/org/spongepowered/api/service/persistence/DataSerializable.java#L38) is supposed to return a **NEW** DataContainer that represents the data of the serializable object, not casting the object as a DataContainer.

2) This cast will fail as none of the extending classes implement DataContainer.